BLSQ/iaso
# WC2-203 Add possible_fields on FormVersion

add data mirgation
expose it on the API

Related JIRA tickets : WC2-203

## Self proofreading checklist

- [NA] Are my typescript files well typed
- [NA] New translations have been added or updated if new strings have been introduced in the frontend
- [NO] Are there enough tests
- [NO] Documentation has been included (for new feature)

## Changes

Add possible_field like we have on form but per version.
Made a migration
Keep it up to date via a signal.

## How to test
Run migrations.
Check in API via dynamics field:
- `http://localhost:8081/api/formversions/?fields=possible_fields&order=-id `

Should roughly match what is in Form
`http://localhost:8081/api/forms/?fields=possible_fields&order=-id`

Add new form version to check it correspond to what you expect.
![image](https://user-images.githubusercontent.com/82500/230390636-80a40a48-9060-44ca-9ae1-c13455a85323.png)
